### ISSUE TITLE
BOP: knowledgebase search: requirements

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/BOP#274]
### Overview
We need to outline the requirements for what searching the knowlegebase so that wireframes can be started

### Details
The wireframes will be the guide by which the development team determines what to build

### Action Items
- [ ] Grab written requirements from kb wiki 
- [ ] outline what the user stories are
- [ ] Add information to the kb requirements doc

### Resources/Instructions
- https://github.com/hackforla/knowledgebase/wiki/Data-Model
- [KB requirements](https://docs.google.com/document/d/1N1yHxnN4pp5x_F93XL3KiVh3tBxcOCUogrFlNl5RJVE/edit#)

**Comment by ag2463** [comment]
### Refined Requirements:

- Title 
- Description
- Practice Area
- Status 
- Contributor(s) 
- Tool 
- Source 
- Topic Area 
- Usability Rating 
- Review
- Date Submitted 
- Asset Type 
- Part number 
- Primary Document ID 
- Original URL 
- Backup URL

**Comment by bonniewolfe** [comment]
### Example 1 (a multiple part process) 
- Record ID: 8
- Title: Project Intake: Refined Intake Process
- Description (this should be changed to description): This slide presentation, is an overview of the intake process for new projects at Code for Boston, and has details about the questions that they ask during due diligence and the information that projects will need to provide in their pitch to CfB members once the project is approved. It has a companion form, that is part of the process.
- Practice Area: Product/Project
- Status: Work in Progress (we will need to clarify what options are available)
- Contributor(s): Harlan, Matt Thad 
- Tools: Process
- Source: Code for Boston
- Topic Area: Drafting a strategic Plan for a project
- Usability Rating: 3
- Date Submitted:  2020-10-01 
- Asset Type: Slide Deck 
- Part number: 1
- Primary Document ID:  8
- Original URL: https://docs.google.com/presentation/d/1_xGjOV2wtqFwj-uxt5mzxJvpZlo8jkCCZgJEb67XyOQ
- Backup URL:  https://docs.google.com/presentation/d/141XdOtkGNxSOY5r_xDZuN2V_ERnCS0B2lVNlhw5tsmg

### Example 2:
- Record ID: 9
- Title: Project Intake - Problem/Project Submission
- Description (should be changed to review): This Google form is part 2 of the Intake process documentation for this brigade
- Practice Area: Product/Project
- Status: Active (we will need to clarify what options are available)
- Contributor(s): Harlan, Matt Thad 
- Tools: Google Form
- Source: Code for Boston
- Topic Area: Drafting a strategic Plan for a project
- Usability Rating: 3
- Date Submitted:  2020-10-01 
- Asset Type: Google Form
- Part number: 2
- Primary Document ID:  8
- Original URL: https://docs.google.com/forms/d/e/1FAIpQLSce0lWiZFMe4HnqgM9iaXDVhpwG06_GNLGKKYR2mcQGlhQ4rQ/viewform
- Backup URL:  https://drive.google.com/file/d/1amvxbXvGbXuZFcQ1jX-LiL4W-N38T5Le

**Comment by bonniewolfe** [comment]
@lizoh12 this issue is where we are starting with the requirements for the kb search/display page.  please see the two comments above for details on what will appear.  The how is of course up to you.

**Comment by bonniewolfe** [comment]
Items with no filter, use Search instead

- title
- description
- Primary Document

**Comment by bonniewolfe** [comment]
You can use the pnp's from this [spreadsheet ](https://docs.google.com/spreadsheets/d/1N0VSDhYyy5WhX_z18Q0RvLlGO29JGGdMxVsD4X3nFYs/edit#gid=186025086)in your mockups

**Comment by bonniewolfe** [comment]
Example 3 (a process with just one part)
Record ID (this does not display): 10
Title: 
Description (this should be changed to review): 
Status: Work in Progress
Contributor(s): 
Tools: Process
Source: 
Topic Area: 
Usability Rating: 
Date Submitted: 2020-10-01??
Asset Type: 
Part number: 1
Primary Document ID (this does not display): 10
Original URL: 
Backup URL:

**Comment by ExperimentsInHonesty** [comment]
Added a copy of [CTJs icons figma](https://www.figma.com/file/G5bOqhud6azbxyR9El9Ygp/Civic-Tech-Jobs?type=design&node-id=3348-32329&mode=design&t=CCym0TjBeKaMtyCK-0) to the [BOP iconography page in figma](https://www.figma.com/file/XEr9WwIWKCVLaUZh5VQhXw/BOP-Design?type=design&node-id=6362-384&mode=design&t=18YjrJbGNJs83bbK-0)

**Comment by ermbrown** [comment]
### **2/5 Notes**

Resources/Instructions:

Civic Tech Jobs
- Use icons in the Practice Area column
- Icons added to the Iconography page
Parts Column
- 4 types: 1/1, 1/2 & 2/2, 5/9 multiple parts
- Link to primary
- brainstorm additional ideas
Title Column
- Link to page
Contributors
- Use Github handle
Asset Type icon
- Update icon
- Could type out name (asset type; stack)
- AT 

Action Items:
- [] Add Civic Tech Jobs icons in the Practice Area column
- [] Update Parts Column; create multiple iterations
- [] Add link to Title Column
- [] Update Contributors column
- [] Update Asset Type icon

**Comment by ermbrown** [comment]
### **2/1 Notes**

Resources/Instructions:

List View
- Primary Link: add name to primary link
- Filter: icon and name
- Date: YYYY-MM

Cards
- Continue to work on small, medium, large cards

Topic Overview Feedback:
- Main boxes are too big 
- Content boxes are too small
- Shrink the box part down and indent
- Stick header

Ranking Topic Difficulty Page
- Difficult lines doesn't seem like an axis
- Look at other charts for inspiration
- Axis label is not clear
- Put difficulty on an access arrow and add below
- Make the arrow wider

Action Items:
- [ ] Create Detailed (full page) view
- [ ] Create Medium Card view
- [ ] Create Small Card view
- [ ] Add Primary link name
- [ ] Filter: icon and name
- [ ] Add hover feature

**Comment by ermbrown** [comment]
### **2/26 Notes**

Resources/Instructions:

Reminder:
- Add Alt text 

Hover
- Add hover above

Status:
- Use words
- Remove dot

Cards
- Links at the bottom
- Taxed: slide deck, topic area
- Update status icon
- Title unlinked
- Right side: topic, status, source
- multi part: links; only use if multi part process

Primary
- Multi parts will have primary (2/4, etc)
- Blank if not multi part
- Add link icon

Multi Part
- Use numbers for parts
- Not enough room for the title

Source:
- Use brigade logo 

Action Items:
- [ ] Hover: continue to work on hovering; add hover above
- [ ] Status: words only; remove dot
- [ ] Multi link: update primary column in list view
- [ ] Cards: taxed items, links, primary, multi parts, brigade logo, status, title unlinked
- [ ] Create verbose example

**Comment by ermbrown** [comment]
### **3/4 Notes**

Resources/Instructions:

Multiple Parts
- Gray out to show selected
- Number; no circle

List View
- Title of list view
- title to detailed page
- Header: Title (Click to see details)

Small Card
- Completed icon - dark/clear
- Source icon - bottom corner
- add tasked items to small card

Logo:
- Not separate; just icon

Hover
- at bottom
- do not cover words

Action Items:
- [x] List View: update link icon; add "click to see details"
- [x] Update hover
- [x] Small card: source icon bottom corner
- [x] Close out other issue

**Comment by ermbrown** [comment]
### **3/25 Notes**

Resources/Instructions:
List View
- center icons (AT, UT, tools, source, PA)
- How to display multiple practice areas

Submitted Date
- Discuss at a later date: changing "submitted" to "last updated"
- Note: Bonnie and Ethan have discussed this topic
- Need to review 

Multiple Parts:
- User test 
- Use the option with all the numbers outlined in a circle
- Create depressed version
- Reference Material Design 3

Cards
- Use Card 4 (small card)
- Icon stand alone; not encompassed in circle
- Reference: https://github.com/codeforboston; - http://civictechindex.org/search-projects


Action Items:
- [ ] List View: center icons,
- [ ] List View: multiple practice areas
- [ ] Multiple Parts: created depressed version
- [ ] Cards: update icons and depressed numbers

**Comment by ermbrown** [comment]
### **4/1 Notes**

Resources/Instructions:
List View
- Filter: remove circle from asset type; match column in list view
- Filter: update primary
- Filter: toolkit Hack for LA website as example
- Practice Area: show more PAs by adding + and number; remove dots
- Practice Area: vertical dots; similar to google
- Bottom justify columns
- Center data columns for AT, UT, PA

List View Order
- Title first; contributors at the end
- Practice Area: show more PAs by adding + and number; remove dots
- Practice Area: vertical dots; similar to google
- Bottom justify columns
- Center data columns for AT, UT, PA

Tools/Source
- Use excel to view examples
- update the ? icon

Cards
- View Material Design for pressed state
- Update Detailed View 2 card; reverse colors
- Multiple Parts: parts & 1 OF 1 or multiple

Action Items:
- [ ] Update filter AT
- [ ] Update columns: bottom justify and center
- [ ] PA: +numbers and vertical dots
- [ ] Check Material Design pressed state
- [ ] Detailed View Card: pressed and unpressed states
- [ ] Small card: Parts & 1 OF 1

**Comment by ermbrown** [comment]
### **4/8 Notes**

Resources/Instructions:
List View
- Filter: update font sizes and spacing
- PA: +2; no circle; link

Header Alignment:
- bottom aligned
- second word (2 word title) and single word (1 word title) should align

Issue:
- add figma link to issue

Pressed chips
- continue to update
- bold font; play with stroke width

Cards
- Use excel document to update card 
- Clarify part one and part two (Asset Type, Tools, Source)
- Place on different Figma page
- Update URLS chip

Action Items:
- [ ] Filter: update font sizes and spacing
- [ ] PA: update +2; add hyperlink
- [ ] Header Alignment: bottom aligned
- [ ] Issue: add figma link to issue
- [ ] Continue to work on pressed chip
- [ ] Cards: check excel document, place on different Figma page

**Comment by ermbrown** [comment]
### **4/15 Notes**

Overview/Feedback:

Topic 1
- Details 1
- Details 2
- Details 3

Topic 2
- Details 1
- Details 2
- Details 3

Figma Link:
- 4.15 Page
- [Figma Link ](https://www.figma.com/file/XEr9WwIWKCVLaUZh5VQhXw/BOP-Design?type=design&node-id=6428%3A34&mode=design&t=ofODVMvyT5rp1yCe-1)


Action Items:
- [ ] TBD
- [ ] TBD
- [ ] TBD

**Comment by ermbrown** [comment]
### **4/22 Notes**

Overview/Feedback:

Topic 1
- Details 1
- Details 2
- Details 3

Topic 2
- Details 1
- Details 2
- Details 3

Figma Link:
- 4.15 Page
- [Figma Link ](https://www.figma.com/file/XEr9WwIWKCVLaUZh5VQhXw/BOP-Design?type=design&node-id=6428%3A34&mode=design&t=ofODVMvyT5rp1yCe-1)


Action Items:
- [ ] TBD
- [ ] TBD
- [ ] TBD

**Comment by ermbrown** [comment]
### **4/22 Notes**

Overview/Feedback:

Topic 1
- Details 1
- Details 2
- Details 3

Topic 2
- Details 1
- Details 2
- Details 3

Figma Link:
- 4.29 Page
- [Figma Link ](https://www.figma.com/file/XEr9WwIWKCVLaUZh5VQhXw/BOP-Design?type=design&node-id=6428%3A36&mode=design&t=IIeyI3Sl8ZiuT3uq-1)


Action Items:
- [ ] TBD
- [ ] TBD
- [ ] TBD

**Comment by ermbrown** [comment]
4/29 Notes

Overview/Feedback:

Topic 1

    Details 1
    Details 2
    Details 3

Topic 2

    Details 1
    Details 2
    Details 3

Figma Link:

    4.22 Page
    Figma Link

Action Items:

TBD
TBD
TBD

**Comment by ermbrown** [comment]
5/6 Notes

Overview/Feedback:

Topic 1

    Details 1
    Details 2
    Details 3

Topic 2

    Details 1
    Details 2
    Details 3

Figma Link:

    4.22 Page
    Figma Link

Action Items:

TBD
TBD
TBD

**Comment by ermbrown** [comment]
### **5/20 Notes**

Overview/Feedback:

Topic 1
- Details 1
- Details 2
- Details 3

Topic 2
- Details 1
- Details 2
- Details 3

Figma Link:
- 5.6 Page
- [Figma Link ](https://www.figma.com/file/XEr9WwIWKCVLaUZh5VQhXw/BOP-Design?type=design&node-id=6428%3A41&mode=design&t=chDe1E8kA7wHD3BZ-1)

Action Items:
- [ ] TBD
- [ ] TBD
- [ ] TBD

**Comment by ermbrown** [comment]
### **5/27 Notes**

Overview/Feedback:

Topic 1
- Details 1
- Details 2
- Details 3

Topic 2
- Details 1
- Details 2
- Details 3

Figma Link:
- 5.6 Page
- [Figma Link ](https://www.figma.com/file/XEr9WwIWKCVLaUZh5VQhXw/BOP-Design?type=design&node-id=6428%3A41&mode=design&t=chDe1E8kA7wHD3BZ-1)

Action Items:
- [ ] TBD
- [ ] TBD
- [ ] TBD

**Comment by ermbrown** [comment]
### **6/17 Notes**

Overview/Feedback:

List View/Filter
- Update AT icons
- Update AT icons in filter
- collapse icon update to arrows; remove hamburger icon
- Adjust filter and title; same level; filter and arrow are parallel with title
- closed just filter
- No icons needed
- add number of filters active
- Use screenshot as example


Post MVP
- Create new issue for Post MVP


Figma Link:
- 6.17 Page
- [Figma Link ](https://www.figma.com/design/XEr9WwIWKCVLaUZh5VQhXw/BOP-Design?node-id=6428-47&t=jsNIvU3oAI7NkeLT-1)


Action Items:
- [X] Update AT icons
- [X] Update AT icons in filter
- [X] Update collapsed filter

**Comment by ermbrown** [comment]
### **MM/DD Notes**

Overview/Feedback:

Topic 1
- Details 1
- Details 2
- Details 3

Topic 2
- Details 1
- Details 2
- Details 3

Figma Link:
- 5.6 Page
- [Figma Link ](https://www.figma.com/file/XEr9WwIWKCVLaUZh5VQhXw/BOP-Design?type=design&node-id=6428%3A41&mode=design&t=chDe1E8kA7wHD3BZ-1)

Action Items:
- [ ] TBD
- [ ] TBD
- [ ] TBD

**Comment by ermbrown** [comment]
### **7.12 Notes**

Overview/Feedback:

Filter
- Update lozenges from the side to the top
- Find examples of >> and update the page
- Update filter to match table

Figma Link:
- 7.11Page
- [Figma Link ](https://www.figma.com/design/XEr9WwIWKCVLaUZh5VQhXw/BOP-Design?node-id=6428-50&t=uU2AJ9AjgFVC5mCV-1)

Action Items:
- [ ] Update Filter
- [ ] Update lozenges

**Comment by ExperimentsInHonesty** [comment]
@ermbrown Here is the list you have been waiting for with notes.  Discuss on Monday

This screenshot comes from [Figma](https://www.figma.com/design/XEr9WwIWKCVLaUZh5VQhXw/BOP-Design?node-id=6731-3554&t=BKWz8Fw9EffwMFkQ-0)
<img src="https://github.com/user-attachments/assets/fee1ffe5-2f7b-4aaf-8bb2-8bf0a2c8ba32" width=250px>

We need to have generic Asset Type Icons and potentially Brand Icons which we are going to call Tools  (see tools list below
GitHub
![image](https://github.com/user-attachments/assets/222e6f17-0bc0-49f8-99c0-fdfe5917f1ff)
Youtube
Google Sheets
Google Docs
Google Form
Google Slides
Notion
Trello
PDF
GitBook
Reveal News (not sure yet on this one)
Microsoft Word

If there is a an icon for a kanban board, that has three columns, I recommend using that instead of the two columns icon
![image](https://github.com/user-attachments/assets/9fa38f56-99df-4c96-85c0-f9574806306f)

Between these two icons, we prefer the one on the right
![image](https://github.com/user-attachments/assets/30d79359-2dfb-48f9-a8c1-1535f53a8e72)

We might want a different icon for video
![image](https://github.com/user-attachments/assets/3266e54b-0b36-4b0e-9d61-3bfcf2f209fb)

**Comment by ExperimentsInHonesty** [comment]
@ermbrown also @ag2463 is going to finish working on this which you will be able to use. 
 https://docs.google.com/spreadsheets/d/1ZyeIDeHnzwjnkfYuhzeUZQo5tYXpbdnF-HKqN8cZy_A/edit?gid=508577007#gid=508577007

**Comment by ExperimentsInHonesty** [comment]
<details><summary>Tool Values - table</summary>

Tool Values
--|
Adobe
Fandom
GitBook
GitHub
Google Docs
Google Drive
Google Form
Google Sheets
Google Slides
Meet up
Notion
Reveal News
Slack
Trello
URL
YouTube
</details> 

<details><summary>Tool Values - copyable list</summary>

```
Tool Values
Adobe
Fandom
GitBook
GitHub
Google Docs
Google Drive
Google Form
Google Sheets
Google Slides
Meet up
Notion
Reveal News
Slack
Trello
URL
YouTube
```
</details> 


<details><summary>Asset Type Values - Table</summary>

Asset Type Values
--|
Article
Code Repository
Document
Folder
Form
Gists
Issue closed
Issue template-markdown
Kanban
PDF
Public Meeting Signups
Slide deck
Spreadsheet
Team Messaging Board
Video
Website
WIKI
</details> 

<details><summary>Asset Type Values - copyable list</summary>

```
Asset Type Values
Article
Code Repository
Document
Folder
Form
Gists
Issue closed
Issue template-markdown
Kanban
PDF
Public Meeting Signups
Slide deck
Spreadsheet
Team Messaging Board
Video
Website
WIKI
```
</details>

**Comment by ermbrown** [comment]
### **10/28 Notes**

Overview/Feedback:

List View
- Outline Card 
- Update Filter

Tools & Asset Types
- [Google Sheets](https://docs.google.com/spreadsheets/d/1ZyeIDeHnzwjnkfYuhzeUZQo5tYXpbdnF-HKqN8cZy_A/edit?gid=306938521#gid=306938521)
- Update Icons
- Use Material Design

Figma Link:
- 11.4 Page
- [Figma](https://www.figma.com/design/XEr9WwIWKCVLaUZh5VQhXw/BOP-Design?node-id=6428-53&t=f0ZhIkIElEzc0ugR-1)

Action Items:
- [x] Outline Card 
- [x] Update Filter
- [x] Update Icons

**Comment by ermbrown** [comment]
### **11/11 Notes**

Overview/Feedback:

Asset Type Icons
- Reviewed and discussed selected icons
- Update icons per feedback
- Present updated icon list next week

Tool Icons
- Need to use logo source
- Present updated icon list next week

Figma Link:
- 11.18 Page
- Figma Link

Action Items:
- [x] Update Asset Type icons
- [x] Update Tool icons

**Comment by ermbrown** [comment]
### **11/18 Notes**

Overview/Feedback:

Icons
- Asset Type/Tools feedback
- Update icons using feedback
- Details 3

Spreadsheet
- Create spreadsheet with Iconography & Imagery information
- Use Hack For LA Credits page for guidance 
- [Credits Page](https://www.hackforla.org/credits/)

Figma Link:
- TBD
- Figma Link

Action Items:
- [x] Update Icon list using feedback
- [x] Create spreadsheet with Iconography & Imagery information

**Comment by ermbrown** [comment]
### **01/27 Notes**

Overview/Feedback:

Material Designs/Symbols
- Investigate difference Material Designs/ Material Symbols

Google Doc
- Use template to update document 
- Add to shared drive

Figma Link:
- [Logo](https://www.figma.com/design/XEr9WwIWKCVLaUZh5VQhXw/BOP-Design?node-id=6428-56&t=PkFfcOUsnqOlFOcn-1)
- [BOP Icons Credits](https://docs.google.com/spreadsheets/d/1ffMF7DGbItMsEbpJ9ua8Th0_gZinZtRVSm6uFtdhSyc/edit?usp=sharing)

Action Items:
- [x] Investigate differences with Material Designs/ Symbols
- [x] Use template to update document 
- [x] Add to shared drive
- [x] Create place for logo in Figma

**Comment by ermbrown** [comment]
### **MM/DD Notes**

Overview/Feedback:

Topic 1
- Details 1
- Details 2
- Details 3

Topic 2
- Details 1
- Details 2
- Details 3

Figma Link:
- 5.6 Page
- Figma Link

Action Items:
- [ ] TBD
- [ ] TBD
- [ ] TBD

**Comment by ermbrown** [comment]
### **4/21 Notes**

Overview/Feedback:

Logo
- Note updated logo in Figma
- Reach out and confirm with Shienny

Knowledge Base Requirements wireframe
- Cards: shift contributors icon to the right
- Filters column: add icons and update column
- add tooltop to one card

Feedback
- Get feedback
- user testing in May
- ask Bonnie to recruit UX designer

Figma Link:
- 4.21 Page
- Figma Link

Action Items:
- [x] Update logo page
- [x] update contributors icon
- [x] update filters column
- [x] user testing

**Comment by ExperimentsInHonesty** [comment]
Usability_Rating provided by interviewee | Description
-- | --
1 | Not at all documented
2 | Skeleton outline
3 | Could be used by another brigade with someone to walk them through the process
4 | Could be used by another brigade with a short summary of the process
5 | Could be used by another brigade without extra consultation or context

**Comment by ExperimentsInHonesty** [comment]
Usability Rating | Short Description
-- | --
1 | No documentation
2 | Basic outline only
3 | Usable with guidance
4 | Usable with a summary
5 | Fully self-explanatory

**Comment by ExperimentsInHonesty** [comment]
Usability Rating | Short Description
-- | --
1 | No documentation
2 | Basic outline only
3 | Requires guidance
4 | Needs summary
5 | Fully self-explanatory

**Comment by ExperimentsInHonesty** [comment]
1 page
1 of 1

2 pages
1 of 2

2 of 2

3 pages
1 of 3

2 of 3

3 of 3

4 pages
1 of 4

2 of 4

3 of 4

4 of 4

5 pages
1 of 5

2 of 5

3 of 5

4 of 5

5 of 5

6 pages
1 of 6

2 of 6

3 of 6

4 of 6

5 of 6

6 of 6

7 pages
1 of 7

2 of 7

3 of 7

4 of 7

5 of 7

6 of 7

7 of 7

8 pages
1 of 8

2 of 8

3 of 8

4 of 8

5 of 8

6 of 8

7 of 8

8 of 8

9 pages
1 of 9

2 of 9

3 of 9

4 of 9

5 of 9

6 of 9

7 of 9

8 of 9

9 of 9

10 pages
1 of 10

2 of 10

3 of 10

4 of 10

5 of 10

6 of 10

7 of 10

8 of 10

9 of 10

10 of 10

**Comment by ExperimentsInHonesty** [comment]
- [ ] 1 Asset PnP
- [ ] 1st Asset, Multi Asset PnP
- [ ] All Assets, Multi Asset PnP

---

- [ ] 1 Part Asset
- [ ] 1st Part, Multi Asset
- [ ] All Parts, Multi Asset

**Comment by ermbrown** [comment]
### **6/2 Notes**

Overview/Feedback:

Researchers
- Do they know how to get less/more results?
- What you think they should be able to do?
- What does the filter mean?

Filter
- Add all parts in each category
- update plus/minus
- update usability rating
- 1 Part Asset
- 1st Part, Multi Asset
- All Parts, Multi Asset

List View
- remove row above card (duplicate)
- update column name to Part Type 
- Update Part column to match filter options

Figma Link:
- 6.2 Page
- Figma Link

Action Items:
- [x] Update Filter
- [x] Update List View
- [x] Update 4/21 Action Items

**Comment by ExperimentsInHonesty** [comment]
example of filter menu

<details><summary>Name</summary>

<img src="https://github.com/user-attachments/assets/807b4daa-95e7-4a8e-bb39-e6c0df8b3da5" width=300px>
</details> 

<details><summary>Date</summary>

<img src="https://github.com/user-attachments/assets/1b046847-8656-412b-bf36-f49df1646f4a" width=300px>
</details> 


<details><summary>When there are only a few choices the search is removed</summary>

<img src="https://github.com/user-attachments/assets/6e275e22-f42d-4404-afc2-347356038ce9" width=300px>
</details>

**Comment by ExperimentsInHonesty** [comment]
https://m3.material.io/components/bottom-sheets/guidelines

**Comment by ermbrown** [comment]
### **6/16 Notes**

Overview/Feedback:

List View
- Move filters to the top header
- Add drop-down menus in column headers
- Update Part column (1/3)
- Research Part Type column

Filter Functionality Updates
- Search capabilities
- Multiple filter support
- Date picker
- refer to images above and [TDM](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=21109-26041&t=IwfeuwIN7RMnbcP7-0)
- [Date picker](https://www.figma.com/design/TTRS2FWXsrymHYpPJL1IdH/Internship-Team-Main-file?node-id=5508-17617)


Mobile Implementation
- Bottom sheet approach for mobile filters
- Verify approach against Material 3 UI guidelines
- Consider different interaction patterns

Figma Link:
- 6/16 page
- Figma Link

Action Items:
- [ ] Update filters 
- [x] Update Part column
- [ ] Research M3 UI guidelines for mobile
- [x] Part Type research
